packages/styles/src/components/pagination.scss

```diff
@@ -40,6 +40,7 @@
       &[aria-label='Last'] {
         > span {
           display: inline-block;
+          color: transparent;
```

Comment:
Why?

Reply:
There are default ">>" arrows in the field from ng-bootstrap and they overlap with our icon.